hawkular-alerts-bus-api/src/main/java/org/hawkular/alerts/bus/api/MetricDataMessage.java

```diff
@@ -86,6 +86,8 @@ public String toString() {
      */

```

Comment:
+1
the package https://github.com/hawkular/hawkular-alerts/tree/master/hawkular-alerts-bus-api was created for that.
I agree that at some point it could go to a commons repo to have more clean maven dependencies.